docs/community/topics/page-metadata.md

```diff
@@ -7,9 +7,20 @@ This is [similar to how the TOML language defines nested configuration](https://
 
 For example, to remove the secondary sidebar, we use a page metadata key like this:
 
+`````{tab-set}
```

Comment:
Yep this is the right way to do it. If you want to I believe that you can indent everything within the directive just to make it easier to read.

Reply:
Now that I get I think we can leave it as it is